src/LandingPage.js

```diff
@@ -8,12 +10,24 @@ class LandingPage extends Component {
         this.state={
 
         }
+        var inputString = '';
```

Comment:
This needs to be a property of the state object

Reply:
fixed, input is now stored in the state object.